### PR TITLE
Get extension location dynamically

### PR DESCRIPTION
**Pull request by makandre**
Signed-off-by: Andrew Mak <makandre@ca.ibm.com>

Following changes make it so that the extension can be installed to any location.

### CODE REVIEW

**Comment by makandre** [comment]
@sghung Could you review please?

File: entrypoint.sh [hunk 1]
```diff
@@ -75,7 +77,7 @@ cd "$ROOT"
 hostWorkspacePath=`$util getWorkspacePathForVolumeMounting $LOCAL_WORKSPACE`
 
 # Export some APPSODY env vars
-export APPSODY_MOUNT_HOME=`/codewind-workspace/.extensions/appsodyExtension/scripts/get-home.sh | xargs $util getWorkspacePathForVolumeMounting`
+export APPSODY_MOUNT_HOME=`$DIR/scripts/get-home.sh | xargs $util getWorkspacePathForVolumeMounting`
 export APPSODY_MOUNT_CONTROLLER="$hostWorkspacePath/.extensions/appsodyExtension/bin/appsody-controller"
```
Comment (sghung) [inline entrypoint.sh]: The APPSODY_MOUNT_CONTROLLER should also be using $DIR

Comment (makandre) [reply entrypoint.sh]: hmmm... that is a very good catch

**Comment by makandre** [comment]
Need to rethink this.

**Comment by makandre** [comment]
@sghung Re-opening this now.  After discussion we need to keep the extension in `/codewind-workspace/.extensions`, however changes here make the name of the folder (i.e. `appsodyExtension or `codewind-appsody-extension`) not hardcoded in the scripts

**Pull request merged**